api/api.go

```diff
@@ -19,6 +19,7 @@ var (
 	ONSRealm               = "Florence publishing platform"
 	Charset                = "UTF-8"
 	NewPasswordChallenge   = "NEW_PASSWORD_REQUIRED"
+	MFASetupChallenge      = "MFA_SETUP"
```

Comment:
This appears to be an unused variable.

Reply:
Yeah, James mentioned that we should think about catering for MFA challenges too. I can take it out for now. Include in a new ticket.